tests/kafkatest/services/performance/producer_performance.py

```diff
@@ -13,10 +13,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from kafkatest.services.performance import PerformanceService
+from kafkatest.services.performance.jmx_mixin import JmxMixin
 
-
-class ProducerPerformanceService(PerformanceService):
+class ProducerPerformanceService(JmxMixin):
```

Comment:
Inherit from both `PerformanceService` and `JmxMixin` (see comment on mixin above)
i.e.
`class ProducerPerformanceService(PerformanceService, JmxMixin)`